test/unit/test_warnings.py

```diff
@@ -0,0 +1,20 @@
+import os
+import subprocess
+import tempfile
+
+from scripts.test.shared import WASM_OPT, WASM_DIS, WASM_SHELL, NODEJS, run_process
+from .utils import BinaryenTestCase
+
+
+class WarningsText(BinaryenTestCase):
+    def test_warn_on_no_passes(self):
+      err = run_process(WASM_OPT + [self.input_path('asyncify-pure.wast'), '-o', 'a.wasm'], stderr=subprocess.PIPE).stderr
```

Comment:
The indentation isn't consistent here.

Reply:
Sorry, what isn't consistent? Your eyes must be better than mine :)

Reply:
Oh, 4 spaces... fixing.